roles/StackStorm.st2chatops/tasks/main.yml

```diff
@@ -198,6 +198,14 @@
   notify:
     - restart st2chatops
 
+- name: daemon-reload
+  become: yes
+  command: systemctl daemon-reload
+  
+- name: enable st2chatops
+  become: yes
+  command: systemctl enable st2chatops
```

Comment:
The service is already enabled by the task below via https://docs.ansible.com/ansible/latest/modules/service_module.html